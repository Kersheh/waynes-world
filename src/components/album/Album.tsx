import React from 'react';

import Button from 'components/button/Button';
import styles from './Album.module.scss';

interface AlbumProps {
  title: string;
  artist: string;
  // id?: string;
  // src?: any; // TODO
}
const Album = ({ title, artist }: AlbumProps) => {
  return (
    <Button styleType="album">
      <div className={styles.album}>
        <div className={styles.art} />

        <div className={styles.tag}>
          <div className={styles.title}>{title}</div>
          <div className={styles.artist}>{artist}</div>
        </div>
      </div>
    </Button>
  );
};

export default Album;
