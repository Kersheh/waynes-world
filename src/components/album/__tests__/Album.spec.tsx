import React from 'react';
import { render } from '@testing-library/react';

import Album from '../Album';

describe('<Album/> component', () => {
  it('should render', () => {
    const { container } = render(
      <Album title="Album Title" artist="Artist" />
    );

    expect(container).toMatchSnapshot();
  });
});
