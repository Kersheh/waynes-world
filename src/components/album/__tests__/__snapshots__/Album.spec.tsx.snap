// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Album/> component should render 1`] = `
<div>
  <button
    class="button album"
    type="button"
  >
    <div
      class="album"
    >
      <div
        class="art"
      />
      <div
        class="tag"
      >
        <div
          class="title"
        >
          Album Title
        </div>
        <div
          class="artist"
        >
          Artist
        </div>
      </div>
    </div>
  </button>
</div>
`;
